jacodb-core/src/main/kotlin/org/jacodb/impl/fs/ByteCodeConverter.kt

```diff
@@ -136,7 +147,7 @@ private fun FieldNode.asFieldInfo() = FieldInfo(
 
 val ClassSource.info: ClassInfo
     get() {
-        return newClassNode(ClassReader.SKIP_CODE).asClassInfo(byteCode)
```

Comment:
This part is dangerous. Initialization speed needs to be checked.

Reply:
This part was returned back as ClassSource.fullAsmNode does the job when we need some extra information (e.g. names of arguments)